### PR TITLE
use rust attribute instead of no_run

### PR DESCRIPTION
**Pull request by adsick**

### CODE REVIEW

**Comment by adsick** [comment]
this makes syntax highlighting work.

**Comment by pacak** [comment]
`Readme.md` is included into `lib.rs` and `cargo test` tries to run them as a part of CI (which is not configured yet :) ). Snipped with main fails. I guess we can use `rust` for the second snipped but the first one needs `no_run` of some sort, not sure if those two can combined.

**Comment by pacak** [comment]
Any thoughts? Can you make this change so `cargo test` passes?

**Comment by adsick** [comment]
I dunno honestly, maybe replace \```rust with \```no_run on "include" action?

**Comment by pacak** [comment]
"include" action is done with `#![doc = include_str!("../README.md")]`, so can't replace there. Can you drop the first change? Still going to be an improvement - I'll merge that.

**Comment by adsick** [comment]
like that? (dunno how to "drop" the first change)

**Comment by pacak** [comment]
> like that?

This will do, thank you.

> (dunno how to "drop" the first change)

https://www.atlassian.com/git/tutorials/rewriting-history

TLDR, I would do something like 

1. edit the file removing the first change
2. `git add -p` to add this removal
3. `git commit --amend` to add this change to the previous commit - making it to contain only one change
4. `git push origin master --force-with-lease` - to replace master in your branch with
